evalai/utils/challenges.py

```diff
@@ -0,0 +1,66 @@
+import os
+import json
+import requests
+
+from click import echo
+
+from pylsy import pylsytable
+
+from evalai.utils.auth import get_headers
+from evalai.utils.urls import urls
+from evalai.utils.common import valid_token
+
+
+API_HOST_URL = os.environ.get("EVALAI_API_URL", 'http://localhost:8000')
+
+
+def get_challenges(url):
+
+    headers = get_headers()
+    response = requests.get(url, headers=headers)
+    response_json = json.loads(response.text)
+
+    if valid_token(response_json):
+
+        column_names = ['ID', 'Challenge Name', 'Short Description']
```

Comment:
Please remove `Short Description` for now. We can add it later on too. :) The terminal output doesn't look good with it. We will add it once we have a good library to print on the leaderboard.